File: java/src/main/java/utils/BrowserSetup.java
package utils; 
 
import org.openqa.selenium.WebDriver; 
import org.openqa.selenium.edge.EdgeDriver; 
import io.github.bonigarcia.wdm.WebDriverManager; 
 
public class BrowserSetup { 
    public static WebDriver getDriver() { 
        WebDriverManager.edgedriver().setup(); 
        WebDriver driver = new EdgeDriver(); 
        driver.manage().window().maximize(); 
        return driver; 
    } 
}

File: python/utils/browser_setup.py
from selenium import webdriver 
from selenium.webdriver.edge.service import Service 
from webdriver_manager.microsoft import EdgeChromiumDriverManager 
 
def get_driver(): 
    service = Service(EdgeChromiumDriverManager().install())  # Correct way to set up the service 
    driver = webdriver.Edge(service=service)  # Pass service to WebDriver 
    driver.maximize_window() 
    return driver